searx/engines/bing_news.py

```diff
@@ -11,14 +11,18 @@
  @parse       url, title, content, publishedDate, thumbnail
 """
 
-from urllib import urlencode
-from urlparse import urlparse, parse_qsl
 from datetime import datetime
 from dateutil import parser
 from lxml import etree
 from searx.utils import list_get
 from searx.engines.bing import _fetch_supported_languages, supported_languages_url
 
+try:
+    from urlparse import urlparse, parse_qsl
+    from urllib import urlencode
```

Comment:
Why don't you import `urlencode` from `searx.url_utils` just like in other engines?